### PR TITLE
describe how to create a notification group

### PR DESCRIPTION
**Pull request by nikomatsakis**
Sample PR. I'm considering moving the list of steps into the compiler-team repository, though, as a template tracking issue. Maybe that'd be better.

### CODE REVIEW

File: src/compiler/notification-groups.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ping people and draw their attention to issues. Notification groups
+are setup so that anyone can join them if they want.
+
+## Creating a notification group
+
+If you'd like to create a notification group, here are the steps.
+First, you want to get approval:
+
+* Propose the group by preparing a [Major Change Proposal][MCP].  If
+  you're group is not analogous to some existing group, it is probably
```
Comment (jonas-schievink) [inline src/compiler/notification-groups.md:13]:
```suggestion
  your group is not analogous to some existing group, it is probably
```

File: src/compiler/notification-groups.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  a good idea to ping compiler team leads before-hand or as part of
+  the MCP.
+* The MCP should specify what Github label will be associated with the
```
Comment (shepmaster) [inline src/compiler/notification-groups.md:16]:
```suggestion
* The MCP should specify what GitHub label will be associated with the
```

File: src/compiler/notification-groups.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Notification groups
+
+The compiler team has a number of notification groups that we use to
+ping people and draw their attention to issues. Notification groups
+are setup so that anyone can join them if they want.
+
+## Creating a notification group
+
+If you'd like to create a notification group, here are the steps.
+First, you want to get approval:
```
Comment (LeSeulArtichaut) [inline src/compiler/notification-groups.md:10]: Maybe explicitly say that the approval must come from the compiler team?
```suggestion
First, you want to get approval from the compiler team:
```

File: src/compiler/notification-groups.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+* Propose the group by preparing a [Major Change Proposal][MCP].  If
+  your group is not analogous to some existing group, it is probably
+  a good idea to ping compiler team leads before-hand or as part of
+  the MCP.
+* The MCP should specify what GitHub label will be associated with the
+  notification group. Often this is an existing label, such as
+  `O-Windows`.
+  
+Once the MCP is seconded, here are the steps to actually create the group.
```
Comment (LeSeulArtichaut) [inline src/compiler/notification-groups.md:20]: Just to be sure, should this whole procedure of filing PRs begin once the MCP is seconded or accepted?

Comment (nikomatsakis) [reply src/compiler/notification-groups.md:20]:
```suggestion
Once the MCP is accepted, here are the steps to actually create the group.
```

good point

**Pull request merged**